2015-07-09 15:23:22.067 [error] <0.133.0> CRASH REPORT Process <0.133.0> with 0 neighbours exited with reason: call to undefined function undefined:start() in gen_server:init_it/6 line 330
2015-07-09 15:23:22.203 [error] <0.132.0> Supervisor cache_sup had child cache_server started with cache_server:start_link() at undefined exit with reason call to undefined function undefined:start() in context start_error
2015-07-09 15:23:22.204 [error] <0.130.0> CRASH REPORT Process <0.130.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,cache_server,{undef,[{undefined,start,[],[]},{cache_server,init,1,[{file,"src/cache_server.erl"},{line,36}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,306}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{cache_app,start,[normal,[]]}} in application_master:init/4 line 133
